app/views/questionnaire.py

```diff
@@ -293,11 +369,18 @@ def submit_answers(routing_path, eq_id, form_type, collection_id):
         return redirect(invalid_location.url(metadata))
 
 
+def store_submitted_responses():
```

Comment:
I think this should be renamed as it reads as if the function is doing the storing rather then returning a boolean

Reply:
Changing to is_submission_stored